tests/system/python/e2e/test_e2e_modbus_c_rtu_pi.py

```diff
@@ -84,13 +84,13 @@ def start_south_north(self, reset_and_start_fledge, add_south, skip_verify_north
                   plugin_lang="C", start_service=False, plugin_discovery_name=PLUGIN_NAME)
 
         if not skip_verify_north_interface:
-            start_north_pi_server_c(fledge_url, pi_host, pi_port, pi_token)
+            start_north_pi_server_c_web_api(fledge_url, pi_host, pi_port, pi_db=pi_db, pi_user=pi_admin, pi_pwd=pi_passwd)
```

Comment:
This change alone is not sufficient, as the tests rely on dependent values further downstream, which may still cause issues.

This tests SKIPPED from nightly runs as "Test requires a connected serial port!!"